demo/app.js

```diff
@@ -502,10 +523,38 @@ const Demo = React.createClass({
     });
   },
 
+  _nextSibling () {
+    console.log('next');
+  },
+
+  _previousSibling () {
+    console.log('previous');
```

Comment:
I just wanted an example callback for the demo without spending a lot of time implementing complicated behavior. I am making these props required in the drawer component to prevent the nav content rendering useless buttons.